src/main/TaskManager.java

```diff
@@ -44,43 +73,50 @@ public boolean update(Task oldTask, Task newTask) {
             return false;
     }
 
-    public List<Task> getCompletedTasks(){
+    public List<Task> getCompletedTasks() {
         // return completed tasks
         List<Task> result = new ArrayList<>();
-        for (Task task: tasks) {
+        for (Task task : tasks) {
             if (task.isCompleted())
                 result.add(task);
         }
         return result;
     }
 
-    public List<Task> getActiveTasks(){
+    public List<Task> getActiveTasks() {
         // return incomplete tasks
         List<Task> result = new ArrayList<>();
-        for (Task task: tasks) {
+        for (Task task : tasks) {
             if (!task.isCompleted())
                 result.add(task);
         }
         return result;
     }
 
-    public void removeAllCompleted(){
+    public void removeAllCompleted() {
         // remove all completed tasks
         List<Task> tasksToRemove = new ArrayList<>();
-        for (Task task: tasks){
+        for (Task task : tasks) {
             if (task.isCompleted())
                 tasksToRemove.add(task);
         }
         tasks.removeAll(tasksToRemove);
     }
 
-    public void checkCompleted(Task task){
+    public void checkCompleted(Task task) {
         // logika surandu task'a ir ji complete'inu
         tasks.get(tasks.indexOf(task)).setCompleted();
     }
 
-    public List<Task> order(){
+    public List<Task> order() {
+
         // rikiuojame pagal prioriteta
         return null;
     }
+
```

Comment:
šitą Enum siūlyčiau dėti į `Task` klasę. Nes rikiuojami task'ai ir tai su jais susijęs funkcionalumas, o ne su `TaskManager`